src/relentless/simulate/hoomd.py

```diff
@@ -1282,6 +1310,29 @@ def act(self, timestep):
                             ),
                         ).toNeighborList()
 
+                        # filter bonds from the neighbor list if they are present
+                        # bond exclusions apply regardless of order, so
+                        # consider both (i,j) and (j,i) permutations
+                        if (
+                            self.rdf_params["exclude"]
+                            and snap.bonds.N != 0
+                            and len(neighbors[:]) > 0
+                        ):
+                            bonds = numpy.vstack(
+                                [self.bonds, numpy.flip(self.bonds, axis=1)],
+                            )
```

Comment:
This is a small optimization suggestion, but given that you need to Cantor pair the bonds now, would it be better to sort the rows of bonds and neighbors so you don't have to duplicate them in reverse order?

It would also be good to put the Cantor pairing function into a private function or method so you can reuse it, and it will be easier to read.